specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/ServerDevOpsAudit.json

```diff
@@ -0,0 +1,277 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-02-02-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/devOpsAuditingSettings/{devOpsAuditingSettingsName}": {
+      "get": {
+        "tags": [
+          "ServerDevOpsAudit"
+        ],
+        "description": "Gets a server's DevOps audit settings.",
+        "operationId": "ServerDevOpsAuditSettings_Get",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "devOpsAuditingSettingsName",
```

Comment:
what happens if your service receives a name different than default? perhaps is better to specify /default which is the common pattern for singletons.

Reply:
In case name is not default, a not found settings error will be returned.
We didn't restrict this to default, so we can extend it to other values if needed in the future.

Reply:
So is your RP restricting any other name than default for now and later you'll relax this restriction? In that case, may I ask why you are forcing "default" as the name when later it can be relaxed?